term/patterns/social-in-content.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Title: Social in content
+ * Slug: term/social-in-content
+ * Categories: text
+ * Block Types: core/social-links
+ */
+<div class="wp-block-group" style="padding-top:0;padding-bottom:0"><!-- wp:columns {"style":{"spacing":{"blockGap":{"top":"0","left":"var:preset|spacing|60"},"margin":{"top":"0","bottom":"0"}}}} -->
+<div class="wp-block-columns" style="margin-top:0;margin-bottom:0"><!-- wp:column {"width":"50px"} -->
+<div class="wp-block-column" style="flex-basis:50px"><!-- wp:social-links {"iconColor":"custom-beige","iconColorValue":"#edece8","iconBackgroundColorValue":"#000","style":{"spacing":{"padding":{"top":"0","bottom":"0"},"margin":{"top":"0","bottom":"0"},"blockGap":{"top":"var:preset|spacing|40"}}},"className":"is-style-social-rounded-icons","layout":{"type":"flex","orientation":"vertical","justifyContent":"center","flexWrap":"nowrap"}} -->
```

Comment:
let's use the preset colors of the theme instead of the hex values. This makes it so if we add style variations, the colors will update when the presets are changed by the variation.

If the theme doesn't have a color palette in theme.json, then that means we should add one!